build/tasks/sass.js

```diff
@@ -36,6 +37,16 @@ var iconFiles = "src/icons/**/*.scss";
 
 var assets = ["src/clarity/img/*.png","src/clarity/img/*.svg"];
 
+/*
+ * Adds the current clarity version to generated css files
+ * Depends on this being in the *.scss files: \/* @echo VERSION *\/
+ * NOTE: for this to work we must generate a loud comment -> /*!
+ */
+var env = require('gulp-env');
+var pjson = require('../../package.json'); // Grab the projects package.json w/ version info
```

Comment:
One thing to note and I'm not sure if its important - when using **util.env.version** substitution only happens when we publish **npm run publish:all**, when I run **npm run build** standalone, **util.env.version** is undefined.